kalliope/tts/marytts/README.md

```diff
@@ -0,0 +1,20 @@
+# MaryTTS
+
+## Synopsis
+
+MaryTTS is an open-source, multilingual Text-to-Speech Synthesis platform written in Java.
+
+## Options
+
+| Parameters | Required | Default  | Choices                           | Comment                                               |
+|------------|----------|----------|-----------------------------------|-------------------------------------------------------|
+| voice      | yes      | None     | e.g. bits1, cmu-bdl, enst-camille | Run ./marytts list to check your installed voices     |
+| locale     | yes      | None     | e.g. de, en_US, fr                | Check http://localhost:59125/locales for installed locales| 
+| host       | no       | localhost|                                   | Host address of your MaryTTS server                   |
+| port       | no       | 59125    |                                   | Port of your MaryTTS server                           |
+
+
```

Comment:
In the documentation, can you add an example showing how to set it up in the setting.yml ? 
like : 
```yaml
text_to_speech:
  - marytts:
      voice: "bits1"
      locale: "fr"
      port: "59126"
```

Reply:
Yes this the 'default_tts' flag too. This is a good idea we should add an example of configuration in each stt and tts.